examples/federated-diffusion/config.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Params:
+    num_inference_steps: int = 100
+    num_clients: int = 2
+    num_epochs: int = 1
+    num_rounds: int = 2
+    device: str = "cpu"
```

Comment:
```suggestion
    server_device: str = "cpu"
```

Reply:
I'd rather rename this so it is clear this should be used to specify the `device` on which the server runs (e.g. to do global evaluation). The device clients use should only be set with `client_resources` when using Flower simulation.